src/components/ApartmentApplicantList/components/ApplicantListItem/index.js

```diff
@@ -132,19 +156,41 @@ export default class ApplicantListItem extends Component {
           </Grid>
         </Grid>
         <ListItemSecondaryAction>
-          {this.props.isPrimaryApplicant ? (
-            <IconButton edge="end" aria-label="remove">
-              <ClearIcon color="disabled" />
-            </IconButton>
-          ) : (
-            <IconButton
-              edge="end"
-              aria-label="remove"
+          <Button
+            aria-controls="applicant-menu"
+            aria-haspopup="true"
+            disabled={this.props.isPrimaryApplicant}
```

Comment:
Josh's comment: This seems to only allow the user to edit a list item for someone who is not the Primary Applicant. Am I reading that right, and, if so, why keep PrimaryApplicants from editing their own info? Maybe I'm confused because I don't know what sort of things are being edited...

Reply:
That prop is in the list item, and each list item represents on of the applicants in the list. I believe that prop indicates whether the person represented by that list item is the primary applicant, not about the user.

I will check the surrounding code in a few minutes to confirm or deny

Reply:
This is indeed the case. 
The reason it is disabled for the primary applicant is that the only options in that edit menu are "Make primary applicant" and "Remove". 
There is no need to enable the "Make primary applicant" button if the person is already the primary applicant, and the user is not allowed to remove the primary applicant because they must assign a new primary applicant before they can be removed from the group. Since those are the only options in that edit menu, and both actions are designed to be impossible to use _on_ the primary applicant, it makes sense to disable the menu containing those options.

Reply:
Okay, that makes sense.